api_tests/btsieve/ethereum_only/test.ts

```diff
@@ -29,12 +28,10 @@ const aliceWallet = new Wallet("alice", {
 const alice_wallet_address = aliceWallet.eth().address();
 
 setTimeout(async function() {
-    describe("Test btsieve API", () => {
+    describe("Test btsieve API - ethereum", () => {
         let token_contract_address: string;
         before(async function() {
-            this.timeout(5000);
-            await bitcoin.ensureSegwit();
-            await tobyWallet.btc().fund(5);
+            this.timeout(10000);
```

Comment:
😄  This does less stuff but takes longer?

Reply:
This just means that the timeout is longer, in the best case this should not do any harm, in the worst case if something crashes, than this will only timeout after 10 sec